decode_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	"time"
 
 	. "gopkg.in/check.v1"
+
```

Comment:
Can you amend this commit with the literal command that you ran to "Fix go vet errors"?

This would be helpful for me and the others on the team who aren't as well learned in Go lang.

Reply:
I haven’t used a command, the fixes were all manual, going over the vet errors one by one.

Reply:
Ah. Thanks for letting me know.